config/locales/translation.de.yml

```
@@ -769,7 +769,7 @@ de:
       und sein Eintrag seit einiger Zeit nicht mehr aktualisiert wurde.
       </p>
       <p>
-      Ihr Best-Practices-Ausweiseintrag befindet sich bei
+      Ihr Best-Practices-Eintrag befindet sich bei
```

Not pretty, but I think we should preserve "badge" here

```suggestion
      Ihr BestBest-Practices-Badge-Eintrag befindet sich bei
```

---

Implemented :)